lib/AST/NameLookup.cpp

```diff
@@ -2201,8 +2277,6 @@ directReferencesForTypeRepr(Evaluator &evaluator,
   }
 }
 
-/// Retrieve the set of type declarations that are directly referenced from
-/// the given type.
 static DirectlyReferencedTypeDecls
 directReferencesForType(Type type) {
```

Comment:
This looks like it could fit on one line

Reply:
Ok. Will do in a follow-up

Reply:
It ended up in https://github.com/apple/swift/pull/18562/files